content/updates/2020-08/2-API-search.md

```diff
@@ -0,0 +1,35 @@
+---
+pim_announcement_img: ./img/api-filters.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New updates to ease API catalog structure synchronization 
+::: meta-data type="New" features="API, Connectivity" available="early August" link-to-doc="https://api.akeneo.com/documentation/filter.html"
+:::
+
+A **new `search` parameter** enables you to add filters when retrieving attributes or families.
+
+Those filters will help reduce the number of data to process and the size of the API call responses. 
```

Comment:
It reduces the number of API calls needed to synchronize systems easing delta synchronization on a list of products.
I don't know if it's more explicit like that :D